CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_pinterest v0.11.0
+[PR #34](https://github.com/fivetran/dbt_pinterest/pull/34) includes the following updates:
+
+## Feature update
+- We have added the following fields to each `pinterest_ads` end model:
```

Comment:
Maybe we can add a bit more fanfare here to highlight native support for conversions.